hijack/locale/ru/LC_MESSAGES/django.po

```diff
@@ -12,6 +12,19 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "X-Generator: Poedit 2.4.2\n"
 
+#: contrib/admin/admin.py:35
+msgid "hijack user"
+msgstr "участник угона"
```

Comment:
```suggestion
msgstr "захватить пользователя"
```

Reply:
Haha, thanks. I used deepl. A friend told me already that it's not super reliable for Russian.

Reply:
Deepl is cool and usually gives good results. 
But in this case it was really funny. A "участник угона" is a person who participated in the theft of a car or an airplane, for example.

But since I am a native russian speaker, our russian translations are safeguarded :)